### PR TITLE
Biaxial_increment

### PR DESCRIPTION
**Pull request by nblomberg**
Just like uniaxial_increment but takes strain increments from 2 axes

### CODE REVIEW

**Comment by jvaara** [comment]
Please create a test for this as well.

**Comment by nblomberg** [comment]
Sure

File: test/test_biaxial_increment.jl [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# This file is a part of JuliaFEM.
+# License is MIT: see https://github.com/JuliaFEM/Materials.jl/blob/master/LICENSE
+using Test, Tensors
+dtime = 0.25
+parameters = IdealPlasticParameterState(youngs_modulus = 200.0e3,
```
Comment (jvaara) [inline test/test_biaxial_increment.jl:5]: I would suggest changing the test material to Chaboche here.

File: test/test_biaxial_increment.jl [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+multiaxial_test = zeros(length(dstrains11))
+for i in 1:length(dtimes)
+    dstrain11 = dstrains11[i]
+    dtime = dtimes[i]
+    biaxial_increment!(mat, dstrain11, dstrain12, dtime)
+    update_material!(mat)
+    push!(stresses, copy(tovoigt(mat.variables.stress)))
+    if i > 1
+        # Check if test reaches plasticity by looking if stresses remain linear
+        plasticity_test[i-1] = !(isapprox(stresses[i+1][1], stresses[i][1]
```
Comment (jvaara) [inline test/test_biaxial_increment.jl:27]: Plasticity can be tracked via internal variable `mat.variables.cumeq` getting non-zero values
https://github.com/JuliaFEM/Materials.jl/blob/master/src/chaboche.jl#L28
https://github.com/JuliaFEM/Materials.jl/blob/master/src/idealplastic.jl#L18

File: test/test_biaxial_increment.jl [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+         + dstrains11[i]/dstrains11[1]*stresses[2][1]; atol=1e-9))
+    end
+    if !(all(x->isapprox(x,0.0;atol = atol=1e-8),tovoigt(mat.variables.stress; offdiagscale=2.0)[2:5]))
+        # Check if test has multiaxial stress
```
Comment (jvaara) [inline test/test_biaxial_increment.jl:31]: We should look that `stress[1]` and `stress[6]` are nonzero and `stress[2:5]` are zero.

**Comment by jvaara** [comment]
Great!

**Pull request merged**